apps/style/applab/style.scss

```diff
@@ -8,10 +8,8 @@
 
 $root: '/blockly/media/applab/'; //TODO: Parameterize for asset pipeline
 
-// ** START OF RULES THAT ARE MODIFIED AT RUNTIME IN adjustAppSizeStyles() **
-
 div#visualization {
-  height: 400px;
+  height: 562.5px;
```

Comment:
I could put formulas in for this I suppose. The scale factor here is 1.25, so this is 450 * 1.25. Do you think these would be better if expressed as a product of a constant and a scale factor?